app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -49,9 +49,7 @@ def show
     end
 
     def available
-      subscriptions = if @system
-                        available_for_system
-                      elsif @activation_key
+      subscriptions = if @activation_key
```

Comment:
Is `def available` an API route and action? the code makes me think it is, but is' missing all other things like docs and what not.